src/index.js

```diff
@@ -1,20 +1,17 @@
 // NOTE: separate `:not()` selectors has broader browser support than the newer
 //  `:not([inert], [inert] *)` (Feb 2023)
-// CAREFUL: JSDom does not support `:not([inert] *)` as a selector; using it causes
-//  the entire query to fail, resulting in no nodes found, which will break a lot
-//  of things... so we have to rely on JS to identify nodes inside an inert container
 const candidateSelectors = [
-  'input:not([inert])',
-  'select:not([inert])',
-  'textarea:not([inert])',
-  'a[href]:not([inert])',
-  'button:not([inert])',
-  '[tabindex]:not(slot):not([inert])',
-  'audio[controls]:not([inert])',
-  'video[controls]:not([inert])',
-  '[contenteditable]:not([contenteditable="false"]):not([inert])',
-  'details>summary:first-of-type:not([inert])',
-  'details:not([inert])',
+  'input:not([inert]):not([inert] *)',
```

Comment:
I still used `:not([inert]):not([inert] *)` instead of `:not([inert], [inert] *)`. The former seems less likely to go wrong in custom CSS selector engines, and anyway I'd expect them to be equivalent in terms of performance.

Reply:
I'm OK with the double `:not` instead of the `:not` list, but I'm not sure we can safely replace the 3 lines you've deleted with the single replacement. `contenteditable` is not an attribute on an `input`; you can put that on any element, e.g. to make a `div` editable. And `details` is not an input field either.